js/mockPhotos.js

```diff
@@ -3,7 +3,7 @@ import {getRandomPositiveInteger, createIdGenerator,getRandomArrayElement} from
 const getCommentId = createIdGenerator();
 const createComments = () => Array.from({length: COMMENTS_COUNT},() => ({
```

Comment:
Добавим случайное количество комментариев от 1 до COMMENTS_COUNT, 1 тоже лучше вынести, как минимальное количество комментариев. Это для того, чтобы было лучше виден результат, иначе везде по 3 комментария и непонятно, всё ли у нас правильно работает
```suggestion
const createComments = () => Array.from({length: getRandomPositiveInteger(1, COMMENTS_COUNT)},() => ({

```

Reply:
+

Reply:
Тут я решил попробовать использовать рандомное количество комментариев и отказаться от использования  переменной COMMENTS_COUNT.